svgnative/CMakeLists.txt

```diff
@@ -1,396 +1,420 @@
+endif()
+
+##############################
+# Library settings
+##############################
+set(svgNative_src
+    ${gl_public}
+    ${gl_deprecated_styling}
+    ${gl_source}
+    ${text_port}
+    ${cg_port}
+    ${skia_port}
+    ${gdiplus_port}
+    ${cairo_port}
+    ${dll_version}
+)
+if (USE_SHARED)
+add_definitions(-DBUILDING_DLL)
+add_library(SVGNativeViewerLib SHARED ${svgNative_src})
+else()
+add_library(SVGNativeViewerLib STATIC ${svgNative_src})
+endif()
+##############################
+# Header search paths
+##############################
+
+target_include_directories(SVGNativeViewerLib PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/../")
+target_include_directories(SVGNativeViewerLib PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/include")
+target_include_directories(SVGNativeViewerLib PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/src")
+if(STYLE)
+target_include_directories(SVGNativeViewerLib PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/../third_party/stylesheet/include")
+endif()
+if (USE_TEXT)
+target_include_directories(SVGNativeViewerLib PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/ports/string")
+endif()
+if (USE_CG)
+target_include_directories(SVGNativeViewerLib PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/ports/cg")
+target_include_directories(SVGNativeViewerLib PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/../third_party/cpp-base64")
+endif()
+if (USE_SKIA)
+target_include_directories(SVGNativeViewerLib PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/ports/skia")
+target_include_directories(SVGNativeViewerLib PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/../third_party/skia/include/codec")
+target_include_directories(SVGNativeViewerLib PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/../third_party/skia/include/config")
+target_include_directories(SVGNativeViewerLib PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/../third_party/skia/include/core")
+target_include_directories(SVGNativeViewerLib PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/../third_party/skia/include/effects")
+target_include_directories(SVGNativeViewerLib PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/../third_party/skia/include/encode")
+target_include_directories(SVGNativeViewerLib PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/../third_party/skia/include/gpu")
+endif()
+if (USE_GDIPLUS)
+target_include_directories(SVGNativeViewerLib PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/ports/gdiplus")
+target_include_directories(SVGNativeViewerLib PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/../third_party/cpp-base64")
+endif()
+if (USE_CAIRO)
+target_include_directories(SVGNativeViewerLib PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/ports/cairo")
+target_include_directories(SVGNativeViewerLib PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/../third_party/cpp-base64")
+target_include_directories(SVGNativeViewerLib PUBLIC "${CAIRO_INCLUDE_DIRS}")
+endif()
+
+##############################
+# Library search paths
+##############################
+set(PRIVATE_LIBS)
+set(REQUIRES)
+set(PRIVATE_REQUIRES)
+set(PORTS_INCLUDES)
+
+if(USE_CG)
+    target_link_libraries(SVGNativeViewerLib "-framework CoreGraphics")

```

Comment:
@mpsuzuki Added CF framework.


Not sure why this file shows up so messed up. Commented on all actual changes of this file.